persistence/src/main/scala/com/mesosphere/usi/persistence/store/PersistenceStore.scala

```diff
@@ -0,0 +1,114 @@
+package com.mesosphere.usi.persistence.store
+
+
+import akka.stream.scaladsl.{Flow, Source}
+import akka.util.ByteString
+import akka.{Done, NotUsed}
+import PersistenceStore.{Node, StoreOp}
+import scala.collection.immutable.Seq
+
+import scala.concurrent.Future
+import scala.util.Try
+
+/**
+  * An interface for a simple persistence store abstraction for the underlying Zookeeper store. Supported are basic
+  * create, read, update, delete and check methods along with transaction operations and Zookeeper specific children
+  * and sync methods.
+  */
+trait PersistenceStore {
```

Comment:
No, no. This storage interface is a detail of the `persistence-zookeeper` module and has nothing to do here.